engine/src/main/java/io/camunda/zeebe/engine/processing/deployment/model/element/ExecutableEscalation.java

```diff
@@ -7,12 +7,14 @@
  */
 package io.camunda.zeebe.engine.processing.deployment.model.element;
 
+import io.camunda.zeebe.el.Expression;
 import org.agrona.DirectBuffer;
 import org.agrona.concurrent.UnsafeBuffer;
 
 public class ExecutableEscalation extends AbstractFlowElement {
 
   private final DirectBuffer escalationCode = new UnsafeBuffer();
+  private Expression escalationCodeExpression;
```

Comment:
Here we will run into the same issue which we have with the error event FEEL expressions. I will let you know when we came to a conclusion and then we can apply it to both these PRs.

Reply:
Please have a look at https://github.com/camunda/zeebe/pull/10972#discussion_r1031263828 on how we should approach this for the error event. We should do the same thing here.